common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/SignInCommandResult.kt

```diff
@@ -39,27 +39,50 @@ sealed interface SignInSubmitPasswordCommandResult: INativeAuthCommandResult
  * @see com.microsoft.identity.common.java.nativeauth.providers.responses.signin.SignInTokenApiResult
  */
 interface SignInCommandResult {
-    data class Complete(val authenticationResult: ILocalAuthenticationResult) :
+    data class Complete(
+        override val correlationId: String? = null,
```

Comment:
Why does the `correlationId` variable has a default value in this class but not other classes in this file?